Connectors/ArcGIS/Speckle.Connectors.ArcGIS3/Utils/MapMembersUtils.cs

```diff
@@ -41,20 +41,18 @@ public List<MapMember> UnpackMapLayers(IEnumerable<MapMember> mapMembersToUnpack
   }
 
   // Gets the layer display priority for selected layers
```

Comment:
Let's update the comment here akin to:
```cs
/// <summary>
/// Sorts the selected mapmembers into the same order as they appear in the Table of Contents (TOC) bar in the file.
/// This is a required step before unpacking layers, because depending on the user selection order, some children layers may appear before their container layer if both the container and children layers are selected.
/// </summary>
public List<MapMember> GetMapMembersInOrder(Map map, IReadOnlyList<MapMember> selectedMapMembers)
  ```